seatunnel-e2e/seatunnel-connector-v2-e2e/connector-jdbc-e2e/connector-jdbc-e2e-part-3/src/test/java/org/apache/seatunnel/connectors/seatunnel/jdbc/JdbcVerticaIT.java

```diff
@@ -33,6 +34,8 @@
 import java.util.List;
 import java.util.Map;
 
+@Disabled(
```

Comment:
Disabled until Vertica image is available, please follow https://github.com/vertica/vertica-containers/issues/64